.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,6 +17,19 @@ Closes #0000, References #0000, etc.
 - ...
 - ...
 
+
+#### Testing
+
+<!--
+How did you verify that this change works
+-->
```

Comment:
Put this on one line, otherwise people might add text inside the comment. Also below.